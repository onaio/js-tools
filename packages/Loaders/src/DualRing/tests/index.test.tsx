import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import React from 'react';
import { DualRing } from '..';

describe('Loaders.DualRing', () => {
  it('renders without crashing', () => {
    const wrapper = mount(<DualRing />);

    expect(toJson(wrapper.find('DualRing'))).toMatchSnapshot();
  });
});
