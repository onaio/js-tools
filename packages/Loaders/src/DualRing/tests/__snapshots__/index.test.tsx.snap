// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loaders.DualRing renders without crashing 1`] = `null`;
